docs/advanceDocumentation.md

```diff
@@ -0,0 +1,24 @@
+# Advance Settings
+
+**Table of Contents**
+
+- [Role based Access Control](#-rbac-settings)
+- [Allow and Deny lists](#-allow--and-deny-list-settings)
+- [Free Pricing](#-free-pricing-settings)
+
+## RBAC settings
+
+- Setup and host the Ocean role based access control (RBAC) server. Follow the instructions in the RBAC repository: https://github.com/oceanprotocol/RBAC-Server
```

Comment:
better to have no URLs as cleartext but always link them up. In this case you could link up e.g. `role based access control (RBAC) server` or `RBAC repository`